packages/edit-site/src/components/global-styles/font-library-modal/font-collection.js

```diff
@@ -309,7 +311,11 @@ function FontCollection( { slug } ) {
 					<Spacer margin={ 4 } />
 
 					{ ! selectedCollection?.font_families && ! notice && (
-						<Spinner />
+						<HStack align="center">
+							<Spacer />
+							<ProgressBar />
+							<Spacer />
+						</HStack>
```

Comment:
Although some implementations like this can be seen in the Font Library modal, I think it is better to avoid using `Spacer` components alone.

If the `Spacer` component has no children, the height of the component will be zero, and the top and bottom margins will visually shift. In the screenshot below, you can see that the margin applied to the `Spacer` component above the progress bar covers the progress bar.


![spacer](https://github.com/WordPress/gutenberg/assets/54422211/52473acc-6c51-46fd-8ca6-8832fc1bd395)

Below is an example of applying appropriate margins.

```javascript
<Spacer marginX={ 4 }>
	<HStack align="center">
		<ProgressBar />
	</HStack>
</Spacer>
```

Reply:
Thanks @t-hamano! I've improved the use of these Spacers with the following in https://github.com/WordPress/gutenberg/pull/60570/commits/c189a7c514308d841b12f6442ff820f52c6186c7, as I'd also like to retain the center alignment:

```
<Spacer paddingY={ 6 }>
      <Flex justify="center" align="center">
	      <ProgressBar />
      </Flex>
</Spacer>
```